javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -19,8 +19,11 @@
 package com.github.javaparser.ast.body;
 
 import com.github.javaparser.ast.AccessSpecifier;
+import com.github.javaparser.ast.Modifiers;
+import com.github.javaparser.ast.NodeWithModifiers;
 
 import java.lang.reflect.Modifier;
```

Comment:
We have to remove that ref to java.lang.reflect. We can't support j8 with reflection constants not up to date!

Here is what I propose:
```
public enum Modifier {
  ABSTRACT, FINAL, NATIVE, PRIVATE, /* ... */;

  // ...
  // Helpers tagada...
}

public interface NodeWithModifiers {
  int getModifiers();
  EnumSet<Modifier> getModifiersSet();
}

public class ModifierSet {

  // For compatibility
  public static final Modifier PUBLIC = Modifier.PUBLIC;

  // ...
  // Helpers tagada...
}
```

Else let's keep integers!